index.html

```diff
@@ -0,0 +1,67 @@
+<!doctype html>
+
+<html lang="en">
+<head>
+	<meta charset="utf-8">
+	<title>Portfolio page</title>
+	<meta name="description" content="Holly's portfolio page">
+	<meta name="author" content="Holly Bedford">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link rel="stylesheet" href="style.css" type="text/css">
+	<link rel="stylesheet" href="https://necolas.github.io/normalize.css/8.0.1/normalize.css" type="text/css">
+</head>
+
+<body>
+	<header>
+		<a class="btn" href="#contacts">Contact</a>
+		<h3>Portfolio page</h3>
+		<a class="btn" href="#items">Portfolio stuff</a>
+		
+	</header>
+	
+	<div class="banner"></div>
+	
+	<main>
+		<p class="quote">"Spends too much time with the cat"</p>
+		<a class="btn" href="#about-me">About me</a>
+	</main>
+	
+	<div class="banner">Portfolio Stuff</div>
+	
+	<section class="items" id="items">
+		<div class="portfolio-item pi-image1">
+			<h5>Podcast editing skills</h5>
+		</div>
+		<div class="portfolio-item pi-image2">
+			<h5>Portfolio wireframe</h5>
+			<p>I used Balsamiq to do the wireframe as it's low-fidelity and all I needed at the time.</p>
+		</div>
+		<div class="portfolio-item pi-image3">
+			<h5>Portfolio item here</h5>
+			<p class="no">At some point there will be another portfolio item here but at the moment there is nothing interesting.<br>Or is there?</p>
```

Comment:
Try to avoid using <br> tags. Try wrapping the second sentence in another <p> tag if you want it to be on it's own line!